BookingsApi/BookingsApi.IntegrationTests/Api/V2/Hearings/BookNewHearingV2Tests.cs

```diff
@@ -264,6 +265,32 @@ public async Task should_return_validation_error_when_booking_a_hearing_with_jud
         validationProblemDetails.Errors["judiciaryPerson"][0].Should()
             .Be(DomainRuleErrorMessages.ParticipantWithJudgeRoleAlreadyExists);
     }
+            
+    [Test]
+    public async Task should_book_a_hearing_without_a_judge()
```

Comment:
Good test! but since the changes are in the domain, this will affect V1 too. I would advise adding a book without a judge test in there too

Reply:
Yeah fair enough, I wasnt sure if mattered dupilicating the tests in V1 if it's only meant to be used in V2, but you're right the v1 endpoint will still return the status so I will add the tests there